src/batch-client/index.ts

```diff
@@ -566,6 +567,92 @@ export class ZimbraBatchClient {
 			return normalized;
 		});
 
+	public searchConversation = ({
```

Comment:
```
public searchConversation = ({
	conversationId,
	query,
	header,
	timezone,
	locale,
	cursor,
	...options
}: SearchConversationOptions) => {
	let body: any;
	body = {
		cid: conversationId,
		...{
			_content: {
				...(query && { query }),
				...(header && { header }),
				...(timezone && { tz: timezone }),
				...(locale && { locale }),
				...(cursor && { cursor })
			}
		}
	};

	each(options, (value, key) => {
		if (value) {
			if (key === 'allowableTaskStatus' || key === 'types') {
				body[key] = value.join(',');
			} else {
				body[key] = value;
			}
		}
	});

	return this.jsonRequest({
		name: 'SearchConv',
		body: body
	}).then(res => {
		const normalized = normalize(SearchResponse)(res);
		if (get(normalized, 'conversations.0.messages')) {
			normalized.conversations[0].messages = normalized.conversations[0].messages.map(
				(m: any) => normalizeMessage(m, this.origin)
			);
		}
		return normalized;
	});
};
```

Reply:
`each` isn't a function, and I can't find a reference to it (in lodash or something similar) in this repo or the zm-x-web one. I tried `Array.prototype.slice.call(options)` (which I found in zm-x-web) but that produces an empty array.

Reply:
https://lodash.com/docs/4.17.10#forEach, each is alias of forEach

import forEach from 'lodash-es/forEach';